pipelines/pipeline_upgrade_nightly.yml

```diff
@@ -0,0 +1,82 @@
+- hosts: localhost
+  vars:
+    forklift_name: pipeline-upgrade-nightly
+    forklift_boxes:
+      pipeline-upgrade-centos7:
+        box: centos7
+        memory: 6144
+  roles:
+    - forklift
+
+- hosts: pipeline-upgrade-centos7
+  become: yes
+  vars:
+    puppet_repositories_version: 4
+    katello_repositories_version: 3.6
+    foreman_repositories_version: 1.17
+  roles:
+    - umask
+    - selinux
+    - etc_hosts
+    - epel_repositories
+    - puppet_repositories
+    - foreman_repositories
+    - katello_repositories
+    - update_os_packages
+    - haveged
+    - disable_firewall
+
+- hosts: pipeline-upgrade-centos7
+  become: yes
+  vars:
+    foreman_installer_scenario: katello
+    foreman_installer_options_internal_use_only:
```

Comment:
Perhaps we should add https://github.com/theforeman/forklift/blob/52da2e3270c694c4e407760f9df6782636bad6d1/pipelines/pipeline_foreman_nightly.yml#L19-L21 to reduce memory usage a bit?

Reply:
I think I added this properly to the end of this.